src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. Consider specifying a generic family as a fallback, e.g., 'Roboto', sans-serif; should be 'Roboto', Arial, sans-serif;